Makefile

```
@@ -261,7 +261,7 @@ lint:
 
 format:
 	@go run github.com/golangci/golangci-lint/cmd/golangci-lint run ./... --fix
-	@go run mvdan.cc/gofumpt -l -w .
+	@go run mvdan.cc/gofumpt -l -w x/ app/ ante/
```

The unfortunate makefile update, gofmt / gofumpt have no way to ignore a directory from the path as far as I can tell

---

`tests/e2e` could also be useful
```suggestion
	@go run mvdan.cc/gofumpt -l -w x/ app/ ante/ tests/e2e
```

---

Great call!